#!/usr/bin/env python
PACKAGE = "cbot_monitoring"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("max_roll", double_t, 0, "Max allowable roll", 10.0 ,  0.0, 90.0)
gen.add("max_pitch", double_t, 0, "Max allowable pitch", 25.0 , 0.0, 90.0)
gen.add("max_depth", double_t, 0, "Max allowable depth", 10.0 , 0.0, 100.0)
gen.add("max_speed", double_t, 0, "Max allowable speed", 1.0 , 0.0, 5.0)
gen.add("safety_flag", int_t, 0, "Safety check flag", 0 , 0, 1)

exit(gen.generate(PACKAGE, "Config", "Safety"))